slackbot.sh

```diff
@@ -29,6 +35,7 @@ else
 fi
 
 # Format and send slack webhook
-format='{"attachments":[{"author_name": "Released by: %s", "title": "%s", "text": "Version %s has been released\n%s"}]}'
-data=$(printf "$format" "$RELEASED_BY" "$PROJECT_NAME" "$tag" "$releaseUrl")
+format='{"attachments":[{"author_name": "Released by: %s", "title": "%s", "text": "Version %s has been released\n%s\n\n%s"}]}'
+data=$(printf "$format" "$RELEASED_BY" "$PROJECT_NAME" "$tag" "$releaseUrl" "$changelog")
```

Comment:
Does the changelog come out presented nicely?  I was a bit concerned that it might come out looking a bit whack due to slack not really supporting markdown.

Reply:
<img width="659" alt="slack-post-example" src="https://user-images.githubusercontent.com/10115916/46956740-f12ca080-d08d-11e8-8c99-5b19bde6af5a.png">